libsddc/libsddc.cpp

```diff
@@ -252,17 +288,41 @@ int sddc_set_adc_random(sddc_t *t, int random)
     return 0;
 }
 
-/* HF block functions */
-double sddc_get_hf_attenuation(sddc_t *t)
+double sddc_get_adc_rate(sddc_t *t)
```

Comment:
i am not sure why we use double as freq while uint64_t should better serve the purpose in the original sddc interface? @hayguen @fventuri